app/src/App.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useAragonApi } from '@aragon/api-react'
+import { Main, Button } from '@aragon/ui'
+import styled from 'styled-components'
+import BN from 'bn.js'
+
+function toBigNumber(value) {
+  /* outputs `value * 10^18` */
+  return (new BN(`${value}`)
```

Comment:
We probably don't need this string coercion; inputs will just give us strings. Unless we're doing some quick maths on numbers that don't require especially precise calculations for a UI (e.g. calculating percentages), we generally keep numbers as either strings or BNs.